packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableDepositRow.tsx

```diff
@@ -222,32 +230,53 @@ export function TransactionsTableDepositRow({
     [l1.network, tx.asset, tx.tokenAddress]
   )
 
+  const customAddressTxPadding = useMemo(
+    () => (isCustomDestinationAddressTx(tx) ? 'pb-11' : ''),
+    [tx]
+  )
+
+  if (!tx.sender || !address) {
```

Comment:
Can this happen? I guess if we fall back to event logs?

Reply:
Same as in the WithdrawalsRow